drivers/display/mb2_display.c

```diff
@@ -0,0 +1,453 @@
+/*
```

Comment:
In fact, there are still some differences, except what you said, but it’s better to separate the common parts.

Reply:
@vanwinkeljan  Since the first generation is a 3x9 LED array, which is all on the same GPIO port, while the second generation uses a 5x5 LED array, which is not on the same GPIO port, I have changed the structure `struct mb_display`, so it is not recommended to split it.

Reply:
I suggest to remove mb display relevant parts and make it into a separate PR, which would introduce a generic LED matrix driver or something like that.

Reply:
Okay